src/button/story.tsx

```diff
@@ -113,6 +113,7 @@ stories.add('icon + text', () => (
   </Button>
 ))
 
+
```

Comment:
Maybe add a `**[BREAKING CHANGE]**` for this one ;)

Reply:
:)